packages/react-portal-compat-context/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "@fluentui/react-portal-compat-context",
-  "version": "9.0.0-alpha.0",
-  "private": true,
+  "version": "9.0.0",
```

Comment:
@Hotell This package will be released as I plan to use it as a dependency for v0 & v8. I also changed it to be `9.0.0` to avoid bumps later. Is it okay or should I keep it in RC?

Reply:
hmm not sure if this is ok. cant it stay as RC ? I know that might create duplicates etc as pre-releases are hard to dedupe

Reply:
Restored in 6da2f7c.